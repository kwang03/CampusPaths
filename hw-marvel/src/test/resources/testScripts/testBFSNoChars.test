#Tests BFS on a graph with no chars

LoadGraph myGraph noChars.csv

FindPath myGraph Char1 Char2