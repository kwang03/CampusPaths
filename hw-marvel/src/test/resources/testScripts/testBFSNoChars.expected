#Tests BFS on a graph with no chars

loaded graph myGraph

unknown: Char1
unknown: Char2